src/theory/sets/theory_sets_private.cpp

```diff
@@ -798,23 +522,27 @@ void TheorySetsPrivate::checkDownwardsClosure( std::vector< Node >& lemmas ) {
                 std::vector< Node > exp;
                 exp.push_back( mem );
                 exp.push_back( mem[1].eqNode( eq_set ) );
-                assertInference( nmem, exp, lemmas, "downc" );
-                if( d_conflict ){
+                d_im.assertInference(nmem, exp, "downc");
+                if (d_state.isInConflict())
+                {
                   return;
                 }
               }else{
                 //use proxy set
-                Node k = getProxy( eq_set );
+                Node k = d_state.getProxy(eq_set);
                 Node pmem = NodeManager::currentNM()->mkNode( kind::MEMBER, mem[0], k );
                 Node nmem = NodeManager::currentNM()->mkNode( kind::MEMBER, mem[0], eq_set );
                 nmem = Rewriter::rewrite( nmem );
                 std::vector< Node > exp;
-                if( ee_areEqual( mem, pmem ) ){
+                if (d_state.areEqual(mem, pmem))
+                {
                   exp.push_back( pmem );
-                }else{
+                }
+                else
+                {
                   nmem = NodeManager::currentNM()->mkNode( kind::OR, pmem.negate(), nmem );
```

Comment:
This function can benefit from the common used  shorthand `NodeManager* nm = NodeManager::currentNM();`

Reply:
I agree, but that part of the code didn't change.  Trying to keep the changes in this PR from encompassing the entire solver :)